lib/plugins/aws/package/compile/events/alexaSmartHome/index.js

```diff
@@ -11,9 +11,25 @@ class AwsCompileAlexaSmartHomeEvents {
       'package:compileEvents': this.compileAlexaSmartHomeEvents.bind(this),
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8024
+    const appIdSchema = {
```

Comment:
Maybe it'll be good to define `awsAlexaEventToken` at https://github.com/serverless/serverless/blob/d070986d8628b2bdc5ac984e02971cd5a01282e2/lib/plugins/aws/provider/awsProvider.js#L143 and rely on that ?

Especially that it looks we will need same reference in `alexaSkill` event